coalib/core/CircularDependencyError.py

```diff
@@ -3,7 +3,10 @@ class CircularDependencyError(Exception):
     An error identifying a circular dependency.
     """
 
-    def __init__(self, node=None):
+    prefix = 'Circular dependency detected'
```

Comment:
No prefix is not intended to be changeable.

Do you mean overwriting RuntimeError.__str__ ?